server_admin/topics/events.adoc

```diff
@@ -1,7 +1,5 @@
 
-== Auditing and Events
+== Configuring auditing to track events
 
-{project_name} provides a rich set of auditing capabilities.  Every single login action can be recorded and stored in
-the database and reviewed in the Admin Console.  All admin actions can also be recorded and reviewed.  There is also a Listener SPI
-with which plugins can listen for these events and perform some action.  Built-in listeners include a simple log file and the ability
-to send an email if an event occurs.
+[role="_abstract"]
+{project_name} includes a suite of auditing capabilities. {project_name} can record every login action and administrator action which you can review in the Admin Console. {project_name} also includes a Listener SPI that listens for events and can trigger actions. Examples of built-in listeners include log files and sending emails if an event occurs.
```

Comment:
How about "You can record every login and administrator action and review those actions in the Admin Console."